docs/usage.md

```diff
@@ -128,133 +146,119 @@ yarn run gulp test:acceptance # Run only acceptance (in-browser) tests on produc
+and apps contained within will be importable from Python running in the containers. 
 
-#### Set environment variables
+For any satellite apps that provide front-end assets that need to be built, 
+you will need to run that step seperately.
 
-Environment variables from your `.env` file are sourced when the python container
-starts and when you access a running container with `./shell.sh` Your shell
-environment variables, however, are not visible to applications running in Docker.
-To add new environment variables, simply add them to the `.env` file, stop compose
-with ctrl-c, and start it again with `docker-compose up`.
+For example, to work on [college-costs](https://github.com/cfpb/college-costs):
 
-#### Get familiar with Docker Compose, and our configuration
-
-docker-compose.yml contains a sort of "recipe" for running the site. Each entry
-in the Compose file describes a component of our application stack (Postgres,
-Elasticsearch, and Python), and either points to a public image on Dockerhub,
-or to a Dockerfile in cfgov-refresh. You can learn a lot more about Compose
-files in [the docs](https://docs.docker.com/compose/compose-file/)
-
-Similarly, a Dockerfile contains instructions for transforming some base image,
-to one that suits our needs. The Dockerfile sitting in the top level of
-cfgov-refresh is probably the most interesting. It starts with
-[the public CentOS:7 image](https://hub.docker.com/_/centos/), and installs
-everything else neccessary to run our Python dependencies and the Django app
-itself.  This file will only be executed:
+```bash
+# Check out into develop-apps:
+cd develop-apps
+git clone https://github.com/cfpb/college-costs
 
-- the first time you run `docker-compose up` (or the first time after you
-re-create the Docker Machine VM)
-- any time you run `docker-compose build`
+# Build the front-end:
+cd college-costs
+./setup.sh
+```
 
-That's why you need to run `docker-compose build` after any changes to
-/requirements/
+### Attach for debugging
 
-There are other compose subcommands you might be interested in. Consider
-[learning about](https://docs.docker.com/compose/reference/overview/)
-`build`, `restarts`, `logs`, `ps`, `top`, and the `-d` option for `up`.
+If you have inserted a [PDB breakpoint](https://docs.python.org/3/library/pdb.html) in your code 
+and need to interact with the running Django process when the breakpoint is reached 
+you can run [`docker attach`](https://docs.docker.com/engine/reference/commandline/attach/):
 
-#### Develop satellite apps
+- Python 2.7: `docker attach cfgov-refresh_python2_1`
+- Python 3.6: `docker attach cfgov-refresh_python3_1`
 
-Check out any apps you are developing into the develop-apps directory. These
-will automatically be added to the
-[PYTHONPATH](https://docs.python.org/2/using/cmdline.html#envvar-PYTHONPATH),
-and apps contained within will be importable from Python running in the
-container.
+When you're done, you can detach with `CTRL-p CTRL-q`.
```

Comment:
```suggestion
When you're done, you can detach with `Ctrl+P Ctrl+Q`.
```

Reply:
The capitalization was taken directly from the Docker documentation 😛 .

Reply:
I saw that, but what are they smoking? 😝 

I did wonder if it implied a Shift that could cause problems, but I feel like most people would ignore the case of the final letter and only add Shift if it's explicitly written. cf. the shortcut keys shown in macOS menus, for example.